### ISSUE TITLE
"@classmethod" decorator suppresses class variables

### ISSUE THREAD

**Issue by jhenhapl** [microsoft/pylance-release#1026]
## Environment data

-   Language Server version: Pylance language server 2021.3.0
-   OS and version: Windows 10
-   Python version (& distribution if applicable, e.g. Anaconda): Python 3.7.2

## Expected behaviour

![](https://image.prntscr.com/image/qP4VT0wIQ0OskkAOUO9wkQ.png)

![](https://image.prntscr.com/image/p1km_iaOSzmwm4D8nRS0OQ.png)

It works as expected without the `@classmethod` decorator.

## Actual behaviour

![](https://image.prntscr.com/image/p_9_WZxnRNivzmhxjzoJVw.png)

![](https://image.prntscr.com/image/ULmZ1lGDRka53MgaYWwbfA.png)

After adding the decorator, the class variable is unrecognized even though calling `Test.test()` still returns `1`.

## Code Snippet / Additional information

```python
class Test:
    var = 1

    @classmethod
    def test(cls):
        return cls.var

print(Test.test()) # prints 1
```

**Comment by erictraut** [comment]
Thanks for the bug report. This will be fixed in the next release.

**Comment by jakebailey** [comment]
This issue has been fixed in version 2021.3.1, which we've just released. You can find the changelog here: https://github.com/microsoft/pylance-release/blob/main/CHANGELOG.md#202131-10-march-2021
